### ISSUE TITLE
Entering invalid DBus path in Custom mode can brick phone

### ISSUE THREAD

**Issue by egasimus** [CODeRUS/sailfishos-lockscreen-player-patch#5]
So this patch kept restarting and pausing my music, and since it didn't have a "disable" option (_hint hint_) I tried disabling it by setting the DBus path field in Custom mode to empty. Before I'd backspaced halfway through it though (backspacing was slow... that should've let me know there's something going on but I blindly continued) the GUI crashed and I got the "rapidly blinking green" LED indication. Reboot didn't help, and since I couldn't be bothered to figure out how to mount my storage via telnet I just reimaged the phone. Lost a couple of nice photos I'd taken which made above average wallpapers. One of them (dried apple slices on blue fabric) was brilliant even. Oh well.

**Comment by CODeRUS** [comment]
patches are very dangerous, now you know it :)

**Comment by egasimus** [comment]
Fair enough :) But now that's one reason why we have bugtrackers for 'em isnt it :)
In other news I had my music player start pausing out of nowhere again. I'm thinking this might have to do with the (very) crappy headset I was using; either way the issue of being able to brick your phone by messing up the Custom field still stands :)

**Comment by CODeRUS** [comment]
you can contribute to module with commit :)
https://github.com/CODeRUS/mpris2control
